src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/NotifyService.kt

```diff
@@ -195,10 +197,22 @@ class NotifyAppointmentService(
             mapOf(
               "ppFirstName" to ppDetails.firstName,
               "referenceNumber" to referral.referenceNumber!!,
-              "sessionUrl" to location.toString(),
+              "sessionUrl" to sessionFeedbackLocation.toString(),
             )
           )
         }
+        AppointmentEventType.SCHEDULED -> {
+          if (event.appointmentType == AppointmentType.SUPPLIER_ASSESSMENT) {
+            emailSender.sendEmail(
+              initialAssessmentScheduledTemplateID, ppDetails.email,
+              mapOf(
+                "ppFirstName" to ppDetails.firstName,
+                "referenceNumber" to referral.referenceNumber!!,
+                "referralUrl" to generateResourceUrl(interventionsUIBaseURL, ppReferralDetailsUrl, referral.id).toString(),
```

Comment:
@andrewburrowsmoj i added this line since your review based on a conversation with D and Leigh.

Reply:
Ok, that makes sense - especially since there is no way to navigate from feedback details to referral details.